packer/scripts/debian/networking.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash -eux
 
 rm /etc/udev/rules.d/70-persistent-net.rules
-mkdir /etc/udev/rules.d/70-persistent-net.rules
```

Comment:
Because this breaks the initramfs rebuild on Debian 9. AFAIK this shouldn't break any issue with older Debian and may actually fix things.